Feature/Player Class

SauroMatheo opened the pull request.

SauroMatheo committed:
feat: Created Parent Player Class

SauroMatheo committed:
feat: Added children player classes

MikkoPet commented on src/Entity/PlayerRandomBullshitGo.php:
```diff
@@ -0,0 +1,9 @@
+
+class PlayerRandomBullshitGo extends Player {
+    public function attack() {
+        $this->$currentAttack =  rand(0, 1);
```
y manque le bool, boolife le

SauroMatheo committed:
fix: Abstract, $this dollar signs, and bool missing

SauroMatheo committed:
fix: Made push require the fields of a turn as arguments

This way, it's harder to screw up and push an invalid array, while cleaning up the code.
